src/hotspot/share/opto/loopopts.cpp

```diff
@@ -66,7 +67,7 @@ Node* PhaseIdealLoop::split_thru_phi(Node* n, Node* region, int policy) {
     return nullptr;
   }
 
-  int wins = 0;
+  SplitWins wins = SplitWins();
```

Comment:
`SplitWins wins` will initialize a `SplitWins` variable using the default constructor, so `= SplitWins()` is unnecessary.

Reply:
Thank you for pointing it out. Will remove.